client/packages/electron/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "@openmsupply-client/electron",
   "productName": "open mSupply",
   "//": "Version is updated from the root package.json when electron is built, see the 'make' script below",
-  "version": "2.2.00-RC3",
+  "version": "2.2.02-RC1",
```

Comment:
Do we need to update this file separately to the main `package.json`?

Reply:
no - this is updated when the desktop is built, really it shouldn't be committed with a version change as it can be confusing.. but it also doesn't affect anything